### DISCUSSION TITLE
Recent changes in Audible downloads

### DISCUSSION THREAD

**Discussion by rishurajgupta** [audiamus/AaxAudioConverter#96] [General]
Hello sir/madam,

I tried to download aax file from audible.in (India version) but it is downloading in .mp4 format. Also for audible android app the downloaded file is in aaxc format. These files format are currently unsupported by the tool.

Is there any plan of including these formats as well ?
In that case I will probably save those downloads till the tool is updated.

Thanks for such an awesome tool.

**Comment by audiamus** [comment]
People tell me that renaming the file type from MP4 to AAX should do the trick. Internally, an AAX file has always been an MP4 file, but with encrypted audio. However, you'll need your Audible activation code which you get when downloading your book with the Audible App for Win10.

AAXC is something different. Still MP4, but a much more advanced encryption. We will not be able to decrypt it if it has been downloaded with the Audible App for Android or iPhone. What we'll have to is to download it ourselves. I am working on AAX Audio Converter 2 which will bring this download feature.

**Reply by rishurajgupta** [reply]
Thanks, That MP4 to AAX renaming trick worked. 
This is sufficient for me as of now.
Looking forward for AAX Audio Converter 2.
All the best and good luck.
